test/test_um2netcdf.py

```diff
@@ -217,3 +220,121 @@ def test_cube_filtering_no_include_exclude(ua_plev_cube, heaviside_uv_cube):
     cubes = [ua_plev_cube, heaviside_uv_cube]
     result = list(um2nc.filtered_cubes(cubes))
     assert result == cubes
+
+
+# cube variable renaming tests
+@pytest.fixture
+def x_wind_cube():
+    fake_cube = PartialCube("var_name", {'STASH': DummyStash(0, 2)}, "x_wind")
+    fake_cube.cell_methods = []
+    return fake_cube
+
+
+UMStash = namedtuple("UMStash",
+                     "long_name, name, units, standard_name, uniquename")
+
+
+@pytest.fixture
+def um_var_empty_std():
+    """Return an empty um_var lookup."""
+    um_var = UMStash("", "", "", "", "")
+    return um_var
+
+
+CellMethod = namedtuple("CellMethod", "method")
+
+
+@pytest.fixture
+def max_cell_method():
+    return CellMethod("maximum")
+
+
+@pytest.fixture
+def min_cell_method():
+    return CellMethod("minimum")
+
+
+def test_rename_cube_var_name_simple(x_wind_cube, um_var_empty_std):
+    # NB: ignores cell methods functionality
+    assert x_wind_cube.var_name == "var_name"  # dummy initial value
+    um2nc.rename_cube_var_name(x_wind_cube, None, simple=True)
```

Comment:
Should this be using the `um_var_empty_std` fixture here in place of `None`?

Reply:
I'll run both `None` and the empty um var through the test to cover both bases.

Reply:
Fixed in https://github.com/ACCESS-NRI/um2nc-standalone/pull/52/commits/4fabc4bb0f20cee4ee343eaa0932745030e58e79.